hack/run-tests.sh

```diff
@@ -67,6 +67,7 @@ function usage(){
   printf "\n--services_image_name_suffix {NAMESPACE}\n\tSet the build image name suffix to append to usual image names."
   printf "\n--services_image_version {VERSION}\n\tSet the services image version."
   printf "\n--data_index_image_tag {IMAGE_TAG}\n\tSet the Kogito Data Index image tag ('services_image_version' is ignored)"
+  printf "\n--trusty_image_tag {IMAGE_TAG}\n\tSet the Kogito Trusty image tag ('services_image_version' is ignored)"
```

Comment:
This is useless here if you don't add anything in `test` package for BDD tests

Reply:
Hi @radtriste , I'm new to the operator, thanks for the tip! I've added some BDD tests, let me know what do you think!